### PR TITLE
Update the Docker Hub account, org and group documentation and images

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit SvenDowideit@home.org.au

the UI's have changed over time :)

@kencochrane @fredlf @jamtur01 @moxiegirl

### CODE REVIEW

File: docs/sources/docker-hub/accounts.md [hunk 1]
```diff
@@ -34,21 +34,52 @@ page.
 
 ## Organizations and groups
 
-Also available on the Docker Hub are organizations and groups that allow
-you to collaborate across your organization or team. You can see what
-organizations [you belong to and add new organizations](
+To help you collaborate across your projects or teams, Docker Hub also
+provides organizations and groups that allow more than one Hub user to work on
+shares repositories.
```
Comment (jamtur01) [inline docs/sources/docker-hub/accounts.md]: shared?

File: docs/sources/docker-hub/accounts.md [hunk 2]
```diff
@@ -34,21 +34,52 @@ page.
...
 
-From within your organizations you can create groups that allow you to
-further manage who can interact with your repositories.
+### Organization groups
+
+Users that are in the `Owners` group for an organization, can create and
+modify the membership of groups.
+
+Unless they are the organization's `Owner`, users can only see groups that they
+are members of.
```
Comment (jamtur01) [inline docs/sources/docker-hub/accounts.md]: odd to end sentence in "of"? - of which they are members?

File: docs/sources/docker-hub/accounts.md [hunk 3]
```diff
@@ -34,21 +34,52 @@ page.
...
 
 ![groups](/docker-hub/hub-images/groups.png)
 
-You can add or invite users to join groups by clicking on the organization and then clicking the edit button for the group to which you want to add members. Enter a user-name (for current Hub users) or email address (if they are not yet Hub users) for the person you want to invite. They will receive an email invitation to join the group.
+### Repository group permissions
+
+Organization groups can then be used to manage who can interact with your
```
Comment (moxiegirl) [inline docs/sources/docker-hub/accounts.md]: This whole section needs to be written largely in the active voice. 
image

For example, the first sentence 

"Use organization groups to manage the users that interact with your repositories."

File: docs/sources/docker-hub/accounts.md [hunk 4]
```diff
@@ -34,21 +34,52 @@ page.
...
+tab. They are also listed below your user name on your repositories page
+and in your account profile.
 
 ![organizations](/docker-hub/hub-images/orgs.png)
 
-From within your organizations you can create groups that allow you to
-further manage who can interact with your repositories.
+### Organization groups
+
+Users that are in the `Owners` group for an organization, can create and
```
Comment (fredlf) [inline docs/sources/docker-hub/accounts.md]: No comma needed after "organization". Simplify to "Users in the `Owners` group..."

File: docs/sources/docker-hub/accounts.md (hunk 3 above)
Comment (fredlf) [inline docs/sources/docker-hub/accounts.md]: Agreed. Avoid passive voice unless it's needed to eliminate agency.  But it's "users who interact"... :-)

File: docs/sources/docker-hub/accounts.md (hunk 1 above)
Comment (fredlf) [inline docs/sources/docker-hub/accounts.md]: I think we need to briefly define what "orgs" and "groups" are.

File: docs/sources/docker-hub/accounts.md (hunk 3 above)
Comment (SvenDowideit) [inline docs/sources/docker-hub/accounts.md]: dammit, it could possibly be that this individual is having some issues with this.

aka - merci :)

Comment (moxiegirl) [inline docs/sources/docker-hub/accounts.md]: http://www.quickanddirtytips.com/education/grammar/who-versus-that

**Comment by kencochrane** [comment]
LGTM

**Comment by fredlf** [comment]
LGTM

**Pull request merged**